tools/lib/glbToI3dm.js

```diff
@@ -0,0 +1,45 @@
+'use strict';
+
+var Cesium = require('cesium');
+var defaultValue = Cesium.defaultValue;
+var defined = Cesium.defined;
+var DeveloperError = Cesium.DeveloperError;
+
+module.exports = glbToI3dm;
+
+/**
+ * Generates a new Buffer representing a i3dm asset.
+ *
+ * @param {Buffer} glbBuffer A buffer containing a binary glTF asset.
+ * @param {Buffer} [featureTableJSONBuffer] A buffer containing the batch table to use for the b3dm asset.
+ * @param {Buffer} [featureTableBinaryBuffer] A buffer containing the accompanying binary batch table section for the b3dm asset.
+ * @param {Buffer} [batchTableJSONBuffer] A buffer containing the batch table to use for the b3dm asset.
+ * @param {Buffer} [batchTableBinaryBuffer] A buffer containing the accompanying binary batch table section for the b3dm asset.
+ * @param {Number} [gltfFormat] The gltf format for the body of the glTF field.
+ * @returns {Buffer} Buffer representing the b3dm asset.
+ */
+function glbToI3dm(glbBuffer, featureTableJSONBuffer, featureTableBinaryBuffer, batchTableJSONBuffer, batchTableBinaryBuffer, gltfFormat) {
+    if (!defined(glbBuffer)) {
+        throw new DeveloperError('glbBuffer is not defined.');
+    }
+
+    batchTableJSONBuffer = defaultValue(batchTableJSONBuffer, new Buffer(0));
+    batchTableBinaryBuffer = defaultValue(batchTableBinaryBuffer, new Buffer(0));
+    featureTableJSONBuffer = defaultValue(featureTableJSONBuffer, new Buffer(0));
```

Comment:
I'll add this requirement. However, I think this should stay here as the last line of defense. Otherwise, we should throw an error if the feature table is undefined.

What would be the better scenario?